packages/pwa-kit-dev/src/configs/jest/jest.config.js

```diff
@@ -10,12 +10,13 @@ module.exports = {
     testURL: 'http://localhost/',
     verbose: true,
     collectCoverage: true,
+    coveragePathIgnorePatterns: ['app/application-extensions', '/node_modules/'],
     // We need access to jsdom globally in tests.
     // jsdom isn't accessible so we need to use this
     // 3rd party test environment wrapper. When we
     // upgrade to jest 28, we can revert back to jsdom.
     testEnvironment: 'jest-environment-jsdom-global',
-    testPathIgnorePatterns: ['node_modules', 'build'],
+    testPathIgnorePatterns: ['node_modules', 'build', 'app/application-extensions'],
```

Comment:
Same idea as above. 

It would also be nice if somehow this kind of stuff was in the extension sdk and not sprinkled in other packages code directly